views/js/qtiCreator/widgets/interactions/mediaInteraction/states/Question.js

```diff
@@ -274,6 +282,9 @@ define([
                         interaction.removeClass('pause');
                     } else if (isFlaAvailable) {
                         interaction.attr('data-autostart-delay-ms', 0);
+                        interaction.attr('data-sequence-repeats', 1);
+                        interaction.attr('data-sequence-delay-between-ms', 0);
+                        interaction.attr('data-sequence-delay-after-ms', 0);
```

Comment:
Not sure how these new properties should behave with `hidePlayer: false`, so just disable them for simplicity.

Reply:
Seems fine. By the way, the inputs can still be changed by up/down buttons while they look disabled. Can you fix them the same way `data-autostart-delay-ms` input is?